test/language/block-scope/syntax/for-in/identifier-let-allowed-as-lefthandside-expression.js

```diff
@@ -1,11 +1,10 @@
 // Copyright (C) Copyright 2011 the V8 project authors. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 13.1
+es6id: 13.6.0.1
 description: >
     for declaration:
-    missing identifier, "let" disallowed as bound name
-negative: SyntaxError
+    identifier "let" allowed as lefthandside expression
 ---*/
```

Comment:
I think you need "noStrict" for this one, though I think you could declare "var let" earlier in the test to make it work in strict mode too.